Analyzer/plugins/Analyzer.cc

```diff
@@ -1468,115 +1468,118 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
     if(!isData) year="";
     int run_number=iEvent.id().run();
     bool usePixel = true;
-    
+    bool reverseProb = false;
+
     //Ias
     float dEdxErr = 0;
     auto dedxSObjTmp =
-        computedEdx(run_number, year, dedxHits, dEdxSF, dEdxTemplates, usePixel, useClusterCleaning, typeMode_ == 5, false, trackerCorrector.TrackerGains,
+        computedEdx(run_number, year, dedxHits, dEdxSF, dEdxTemplates, usePixel, useClusterCleaning, reverseProb, false, trackerCorrector.TrackerGains,
```

Comment:
why would this be true for the cosmics only analysis?

Reply:
Idk anything about the typeMode 5 analysis, sorry.